kentikapi/client.go

```diff
@@ -161,8 +158,28 @@ func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error
 	)
 }
 
-func makeTLSOption(c Config) grpc.DialOption {
-	if c.DisableTLS {
+func makeGRPCHostPort(apiURL string) (grpcHostPort string, tlsEnabled bool, err error) {
+	tlsEnabled = false
+	u, err := url.Parse(apiURL)
+	if err != nil {
+		return apiURL, tlsEnabled, err
```

Comment:
```suggestion
		return "", false, err
```
It's better to return zero values of parameters in such case instead of arbitrary ones, like `apiURL`

Reply:
Done